workspaces/badges/plugins/badges-backend/src/service/router.ts

```diff
@@ -50,11 +48,9 @@ export interface RouterOptions {
   catalog?: CatalogApi;
   config: Config;
   discovery: PluginEndpointDiscovery;
-  tokenManager: TokenManager;
   auth?: AuthService;
   httpAuth?: HttpAuthService;
   logger: LoggerService;
-  identity: IdentityApi;
```

Comment:
You'll want to keep these around as optional options, but then not pass them in through the wiring for the new backend system. The options can the completely go away whenever we stop exporting `createRouter`

Reply:
Thanks, made these changes, can you give it another look, please?